### PR TITLE
fix(workspaces): add and arrange missing env vars

### PR DESCRIPTION
**Pull request by vapurrmaid**
Need to fill in the details of a few of them. Arranged them to match the output given by 

```
env | grep CODER_
```

env-vars

### CODE REVIEW

File: workspaces/variables.md [hunk 1]
```diff
@@ -42,25 +54,29 @@ env | grep CODER_
         <td>The memory limit given to your workspace in GB</td>
     </tr>
     <tr>
-        <td><code>CODER_IMAGE_TAG</code></td>
-        <td>The image tag used to create your workspace</td>
+        <td><code>CODER_AGENT_TOKEN</code></td>
```
Comment (jawnsy) [inline workspaces/variables.md:57]: IMHO this one doesn't need to be documented (or should only be documented as "for internal use") as this is an implementation detail and not something that people should rely on as being part of API

Comment (vapurrmaid) [reply workspaces/variables.md:57]: got it!

File: workspaces/variables.md [hunk 2]
```diff
@@ -41,6 +54,10 @@ env | grep CODER_
         <td><code>CODER_MEMORY_LIMIT</code></td>
         <td>The memory limit given to your workspace in GB</td>
     </tr>
+    <tr>
+        <td><code>CODER_SHELL</code></td>
+        <td>The default</td>
```
[the remark this answers is not in the document]
Comment (vapurrmaid) [reply workspaces/variables.md:59]: yikes, I didn't push up.

File: workspaces/variables.md [hunk 3]
```diff
@@ -33,6 +33,19 @@ env | grep CODER_
         <td><code>CODER_USERNAME</code></td>
         <td>Your user name</td>
     </tr>
+    <tr>
+        <td><code>CODER_URL</code></td>
+        <td>The base URL of Coder</td>
```
Comment (khorne3) [inline workspaces/variables.md:38]:
```suggestion
        <td>The base URL of your Coder deployment</td>
```

File: workspaces/variables.md [hunk 4]
```diff
@@ -33,6 +33,19 @@ env | grep CODER_
...
+    </tr>
+    <tr>
+        <td><code>CODER_WP_NAME</code></td>
+        <td>The name of the workspace provider hosting the workspace</td>
+    </tr>
+    <tr>
+        <td><code>CODER_ASSETS_ROOT</code></td>
+        <td>The directory where coder adds coder-specific assets during
```
Comment (khorne3) [inline workspaces/variables.md:46]:
```suggestion
        <td>The directory where Coder adds Coder-specific assets during
```

File: workspaces/variables.md [hunk 5]
```diff
@@ -33,6 +33,19 @@ env | grep CODER_
...
+            workspace creation, such as the coder-cli binary</td>
```
Comment (khorne3) [inline workspaces/variables.md:47]:
```suggestion
            workspace creation, such as the `coder-cli` binary</td>
```

File: workspaces/variables.md [hunk 6]
```diff
@@ -49,18 +62,14 @@ env | grep CODER_
...
-    </tr>
     <tr>
         <td><code>CODER_RUNTIME</code></td>
         <td>The container runtime used to start the workspace (either
         `kubernetes/default` or `kubernetes/sysbox` if the workspace
         is a CVM</td>
     </tr>
+    <tr>
+        <td><code>CODER_IMAGE_URI</code></td>
+        <td>The URI for the image used to build the workspace</td>
```
Comment (khorne3) [inline workspaces/variables.md:73]:
```suggestion
        <td>The URI of the image used to build the workspace</td>
```

File: workspaces/variables.md (hunk 5 above)
Comment (vapurrmaid) [reply workspaces/variables.md]: pushed in https://github.com/cdr/docs/pull/643/commits/dc86f04babd66f9d8743a0d3f139248bfb6715bb - will need to check preview

**Comment by vapurrmaid** [comment]
Thanks for the added commits @khorne3 🎉 !!

**Pull request merged**